src/util/calculation-helpers.ts

```diff
@@ -0,0 +1,16 @@
+export function radiansToDegrees(radians: number) {
+    var pi = Math.PI;
```

Comment:
nit: avoid var, also we could just use `Math.PI` directly since the formula is so simple

Reply:
You're right! Thanks!